app/src/main/assets/html/error/secure-connection-failed.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="en">
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Secured Connection Failed</title>
```

Comment:
I understand to land these, we'll first have only English translations. is there a plan for adding support for localisation (l10n) and internationalisation (i18n)? (I can follow an issue and help with that.)